app/views/shared/_resources_table.html.haml

```diff
@@ -3,8 +3,23 @@
     %tr
       %th #
       - columns.each do |column|
-        %th
-          = resource_class.human_attribute_name(column.to_sym)
+        %th.th-sortable
+          - if (defined? unsortable_columns) && (column.in? unsortable_columns)
```

Comment:
Here don't no use `defined?` you can use `Array(local_assigns[:unsortable_columns])` instead.

e.g. 

```ruby
-if column.in?(Array(local_assigns[:unsortable_columns]))
```